### PR TITLE
Add Upcoming Invoices related methods

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

**Comment by XaosSintez** [comment]
retest this please

**Comment by XaosSintez** [comment]
retest this please

**Comment by XaosSintez** [comment]
test this please

File: src/Services/SubscriptionService.php [hunk 1]
```diff
@@ -143,4 +143,36 @@ public function issueInterimInvoice($subscriptionId, $data)
...
         );
     }
+
+    /**
+     * @param string $subscriptionId
+     * @param array|ArrayObject $params
+     *
+     * @return Invoice[]|Collection
+     */
+    public function searchUpcomingInvoices($subscriptionId, $params = [])
```
Comment (2e3s) [inline src/Services/SubscriptionService.php]:
```suggestion
    public function getUpcomingInvoices($subscriptionId, $params = [])
```
There is no search, actually...

**Pull request merged**